src/3.2.0/wso2apim.js

```diff
@@ -200,10 +200,12 @@ async function constructAPIDef(user, gatewayEnv, apiDef, apiId) {
       securityScheme.push('mutualssl');
       securityScheme.push('mutualssl_mandatory');
     }
-    if(apiDef.securityScheme && apiDef.securityScheme.oauth2 && apiDef.securityScheme.oauth2.enabled === false) {
-      //do nothing
-    } else {
-      securityScheme.push('oauth2');
+    if (apiDef.securityScheme && apiDef.securityScheme.oauth2 && apiDef.securityScheme.oauth2.enabled === true) {
```

Comment:
This condition in now inversed from what it used to be, before it was ignoring this case but now line 206 was moved to inside this if. Is this the intended behavior?

Reply:
Yes I inverted it and it should not be like this what it was before